.github/workflows/pr-build.yaml

```diff
@@ -231,12 +231,18 @@ jobs:
         # We use a new `no_unique_dist_dir` feature, which is not yet released
         run: go install github.com/goreleaser/goreleaser@b53dbb89d02aa3673782f3d063d7d5039446baac
```

Comment:
BTW as we have new GoReleaser release with the feature we require (v0.173.2), maybe we can use it instead of using a specific commit on main branch?

Reply:
it' a different story, but I will update it here :)